src/os/bluestore/BlueFS.cc

```diff
@@ -18,6 +18,19 @@
 #undef dout_prefix
 #define dout_prefix *_dout << "bluefs "
 using TOPNSPC::common::cmd_getval;
+/*
+  dout priority hierarchy
+  1 - initializations in/out, any critical error
+  2 - initialization results
+  3 - allocation failures
+  5 - init/cleanup logs
```

Comment:
Not sure I understand what's covered with this class

Reply:
The idea is to give separate actions/results that are part of initialization: eg. _replay, so it would be possible to have debug levels that allow to verify that everyting is initialized OK, while not spamming logs with anything of mundane operations.